configs/config_sandbox/tf_version_debug/dataset-small-3class_tfversion_debug.yaml

```diff
@@ -0,0 +1,33 @@
+dataset_config:
+  dataset_split:
+    train: [
+      "THIN_REF_S2_P1_L3_2496_1563_2159"
+    ]
+    validation: [
+      "THIN_CNT_S2_P1_L4_2334_1578_2159",
+    ]
+    test: [
+      "8bit_AS4_S2_P1_L6_2560_1750_2160",
+    ]
+  stack_downsampling:
+    type: 'linear' # 'None', 'random', 'linear', 'from_start', 'from_end'
+#    frac: 1.0  # if 'random', 'linear', 'from_start', or 'from_end' selected; ignored if `None` selected
+    number_of_images: 100  # if 'random', 'linear', 'from_start', or 'from_end' selected; ignored if `None` selected
+    num_skip_beg_slices: 50 # trims n slices off of beginning of stack with N total slices. Slice n+1 becomes new Slice 1
+    num_skip_end_slices: 50 # trims m slices off of end of stack with N total slices. Slice N-(m+1) becomes new last slice
+  target_size: [512, 512]  # width, height
+  image_cropping:
+    type: 'class' # 'None' (downscale to target), 'linear' (def # crops), 'random' (def # crops), `class` (def # crops), 'all' (all crops)
+    num_per_image: None # if 'linear' (translation) or 'random' selected, then is num of crops (of target size) per image
```

Comment:
should be `Null`

Reply:
ok, I didn't know that, done

Reply:
yeah, if u look into `prepare_dataset.py`, turns out we're just doing string comparisons with `=='None'` for type, but if a relevant type is selected, then a number needs to appear here. if linear or random types are not selected, then this value doesn't even matter